ramp-engine/ramp_engine/cli.py

```diff
@@ -18,6 +19,33 @@ def main():
     pass
 
 
+@main.command()
+@click.option("--config", default='config.yml', show_default=True,
+              help='Configuration file in YAML format containing the database '
+              'information.')
+@click.option("--events-dir", show_default=True,
+              help='Directory where the event config files are located.')
+@click.option('-v', '--verbose', count=True)
+def daemon(config, events_dir, verbose):
```

Comment:
Note for review: This is a bit tricky to test because we can only kill it with a keyboard interruption